storage_service/locations/tests/test_package.py

```diff
@@ -328,12 +328,16 @@ def test_extract_file_aip_from_compressed_aip(self):
         assert os.path.join(output_path, "manifest-md5.txt")
 
     def test_run_post_store_callbacks_aip(self):
-        uuid = "0d4e739b-bf60-4b87-bc20-67a379b28cea"
+        uuid = "473a9398-0024-4804-81da-38946040c8af"
         aip = models.Package.objects.get(uuid=uuid)
         with mock.patch("locations.models.Callback.execute") as mocked_execute:
             aip.run_post_store_callbacks()
             # Only `post_store_aip` callbacks are executed
             assert mocked_execute.call_count == 1
+            # Placeholders replaced in URI and body
+            url = "http://consumer.com/api/v1/aip/%s/" % uuid
+            body = '{"name": "tar_gz_package"}'
```

Comment:
Is there a reason why you didn't test the UUID placeholder?

Reply:
The UUID placeholder is being tested in the URL (the placeholder uri is `""http://consumer.com/api/v1/aip/<package_uuid>/"`) but you're right it would be better to test in the body as well.